test/async-hooks/testcfg.py

```diff
@@ -3,4 +3,4 @@
 import testpy
 
 def GetConfiguration(context, root):
-  return testpy.AsyncHooksTestConfiguration(context, root, 'async-hooks')
+  return testpy.ParallelTestConfiguration(context, root, 'async-hooks')
```

Comment:
This should be changed back now, no? I was under the impression this change was just so you could stress test it.